CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_pinterest_source v0.8.0
+[PR #25](https://github.com/fivetran/dbt_pinterest_source/pull/25) introduces the following changes:
+## Pinterest API v5 Updates
+- Following Pinterest Ads deprecating the v4 API on June 30, 2023 in place of v5, the Pinterest Ads Fivetran connector now leverages the Pinterest v5 API. The following fields have been deprecated/ introduced:
+
+| **Model** | **Removed**  | **New**   |
+|---|---|---|
+|  [stg_pinterest_ads__advertiser_history](https://fivetran.github.io/dbt_pinterest_source/#!/model/model.pinterest_source.stg_pinterest_ads__advertiser_history) | `billing_profile_status`, `billing_type`, `merchant_id`, `owner_user_id`, `status`  | `owner_username`, `permissions`  |
+|  [stg_pinterest_ads__campaign_history](https://fivetran.github.io/dbt_pinterest_source/#!/model/model.pinterest_source.stg_pinterest_ads__campaign_history) |`campaign_budget_optimization_enabled`  | `default_ad_group_budget_in_micro_currency`, `is_automated_campaign`, `is_campaign_budget_optimization`, `is_flexible_daily_budgets`  |
+|  [stg_pinterest_ads__ad_group_history](https://fivetran.github.io/dbt_pinterest_source/#!/model/model.pinterest_source.stg_pinterest_ads__ad_group_history) |  | `pacing_delivery_type`, `placement_group`, `summary_status`, `ad_account_id` |
+
+
+- In the v5 upgrade, `advertiser_id` has been replaced by `ad_account_id`. However, to keep our Pinterest Ads package standard with our other ad packages, we have kept it as `advertiser_id`.
```

Comment:
Is this still true? Maybe it would be worthwhile to include how we renamed `ad_account_id` for the pin promotion history and ad group models to be `advertiser_id` to be inline with the other naming conventions of the field across the schema/package.

Reply:
Good point, forgot to update this

Reply:
Small note, was `advertiser_id` replaced in the `ad_group_history` and `pin_promotion_history`? I was under the impression it didn't even exist previously and was just added as `ad_account_id`? If that is the case, can you reword this to be in line with this understanding.